script.js

```diff
@@ -109,18 +109,21 @@ function drawCarouselSlideLeft() {
         step = slider.length - 1;
     }
 };
-function changesButtonsStatusDisable(disable) {
-    if (disable === true) {
-        buttonRight.disabled = true;
-        buttonLeft.disabled = true;
-    }
-    else {
-        buttonRight.disabled = false;
-        buttonLeft.disabled = false;
+
+function throttle (callback, limit) {
+    var waiting = false;                     
```

Comment:
Use const or let, var old and no longer used.

Reply:
ok